minify-html/mod.ts

```diff
@@ -0,0 +1,109 @@
+import { init, minify as minifyHTML } from "./deps.ts";
+import { merge } from "lume/core/utils.ts";
+import { Exception } from "lume/core/errors.ts";
+
+import type { Page, Site } from "lume/core.ts";
+
+export interface Options {
+  /** The list of extensions this plugin applies to. */
+  extensions: string[];
+  /** If enabled, HTML files will be minified. */
+  minifyHTML: boolean;
```

Comment:
This option is not needed. You have the `extensions` option to configure it.

Reply:
Alright I will remove it.